packages/contracts-bedrock/scripts/go-ffi/differential-testing.go

```diff
@@ -369,23 +369,23 @@ func DiffTestUtils() {
 		checkErr(err, "Error decoding addr")
 		insn, err := strconv.ParseUint(args[2], 10, 32)
 		checkErr(err, "Error decoding insn")
-		mem.SetUint32(uint32(pc), uint32(insn))
+		mem.SetWord(uint32(pc), uint32(insn))
```

Comment:
Should these be:
```suggestion
		mem.SetWord(arch.Word(pc), arch.Word(insn))
```
?

Reply:
Indeed. That's cleaner. Fixed.